gdk-build.sh

```diff
@@ -2,10 +2,14 @@
 
 set -e
 
-mkdir -p greengrass-build/artifacts/aws.greengrass.clientdevices.mqtt.Moquette/NEXT_PATCH
+# Extract from gdk-config.json
+VERSION=`jq -r '.component."aws.greengrass.clientdevices.mqtt.Moquette".version' gdk-config.json`
+
+mkdir -p greengrass-build/artifacts/aws.greengrass.clientdevices.mqtt.Moquette/${VERSION}
```

Comment:
gdk should create both these dirs automatically 

https://docs.aws.amazon.com/greengrass/v2/developerguide/gdk-cli-configuration-file.html

Reply:
not if you use a custom build

Reply:
Really? I don't think I ever created these directories, but GDK was working.... but I suppose you ought to know, you're trying it out right now.

Reply:
Hmmm, ok I stand corrected. Updated PR to remove it.

I swear this used to not work. Either I was on crazy pills or it was a bug which has since been fixed.